Jenkinsfiles/build.groovy

```diff
@@ -106,77 +107,101 @@ if (!run_pipeline) {
     return
 }
 
-node {
-    stage("Build ${COMPONENT_NAME}") {
-      script {
-        openshift.withCluster() {
-          openshift.withProject() {
+// create NodeJS pod to run verification steps
+def nodejs_label = "jenkins-nodejs-${UUID.randomUUID().toString()}"
+podTemplate(label: nodejs_label, name: nodejs_label, serviceAccount: 'jenkins', cloud: 'openshift', containers: [
+    containerTemplate(
+        name: 'jnlp',
+        image: '172.50.0.2:5000/openshift/jenkins-slave-nodejs:8',
+        resourceRequestCpu: '500m',
+        resourceLimitCpu: '1000m',
+        resourceRequestMemory: '1Gi',
+        resourceLimitMemory: '2Gi',
+        workingDir: '/tmp',
+        command: '',
+        args: '${computer.jnlpmac} ${computer.name}'
+    )
+])
+{
+    node (nodejs_label) {
+        checkout scm
+        dir(CONTEXT_DIRECTORY) {
+            try {
+                sh '''
+                    node -v
+                    npm install
+                '''
+                /* TODO - these do not run correctly in pipeline
```

Comment:
Do we need these if the tests were run in GHA?

Reply:
I was wondering that myself. The older pipeline was running these so I just kept it the same

Reply:
I'm thinking this code isn't needed (even commented out). We have GH history if we ever need to restore it.

Reply:
updated